src/scripts/main.js

```diff
@@ -1,3 +1,424 @@
+  'San Francisco',
+];
+
+function createInput(nameOfInput, type) {
+  const formInputTitle = document.createElement('label');
+  const formInput = document.createElement('input');
+
+  formInputTitle.innerText = nameOfInput[0]
+    .toUpperCase() + nameOfInput.slice(1) + ':';
+  formInput.name = nameOfInput;
+  formInput.type = type;
+  formInput.required = true;
+
+  formInput.setAttribute('data-qa', nameOfInput);
+
+  formInputTitle.append(formInput);
+
+  return formInputTitle;
+}
+
+function createSelect(nameOfSelect, options) {
+  const formInputTitle = document.createElement('label');
+  const formSelect = document.createElement('select');
+
+  formSelect.name = nameOfSelect;
+
+  formInputTitle.innerText = nameOfSelect[0]
+    .toUpperCase() + nameOfSelect.slice(1) + ':';
+
+  options.forEach(option => {
+    const selectOption = document.createElement('option');
+
+    selectOption.value = option;
+    selectOption.innerText = option;
+
+    formSelect.append(selectOption);
+  });
+
+  formSelect.required = true;
+  formSelect.setAttribute('data-qa', nameOfSelect);
+
+  formInputTitle.append(formSelect);
+
+  return formInputTitle;
+}
+
+const inputName = createInput('name', 'text');
+const inputPosition = createInput('position', 'text');
+const inputAge = createInput('age', 'number');
+const inputSalary = createInput('salary', 'number');
+const selectOffice = createSelect('office', cityesOfOffice);
+const buttonSumbit = document.createElement('button');
+
+buttonSumbit.type = 'submit';
+buttonSumbit.innerText = 'Save to table';
+
+const formFields = [
+  inputName,
+  inputPosition,
+  selectOffice,
+  inputAge,
+  inputSalary,
+  buttonSumbit,
+];
+
+function constructForm(fieldsOfForm, className) {
+  const newForm = document.createElement('form');
+
+  newForm.className = className;
+
+  fieldsOfForm.forEach(formField => {
+    newForm.append(formField);
+  });
+
+  return newForm;
+}
+
+const form = constructForm(formFields, 'new-employee-form');
+
+document.body.append(form);
+
+// Notifications
+
+const pushNotification = (
+  title,
+  description,
+  type,
+  posTop = 10,
+  posRight = 10,
+) => {
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', `${type}`);
+  notification.style.boxSizing = 'unset';
+  notification.style.right = `${posRight}px`;
+  notification.style.top = `${posTop}px`;
+
+  notification.setAttribute('data-qa', 'notification');
+
+  const notificationTitle = document.createElement('h2');
+  const notificationDescription = document.createElement('p');
+
+  notificationTitle.className = 'title';
+  notificationTitle.innerText = `${title}`;
+  notificationDescription.innerText = `${description}`;
+
+  notification.append(notificationTitle, notificationDescription);
+
+  document.body.append(notification);
+
+  const removeNotification = () => {
+    notification.remove();
+  };
+
+  setTimeout(removeNotification, 3000);
+};
+
+// Form validation
```

Comment:
it is ok, but in the future, you can make validation logic shorter by using `FormData` object. Read about it.